syncd/Syncd.cpp

```diff
@@ -2484,27 +2553,42 @@ void Syncd::processFlexCounterEvent( // TODO must be moved to go via ASIC channe
     auto groupName = key.substr(0, delimiter);
     auto strVid = key.substr(delimiter + 1);
 
+    auto effective_op = op;
+
     sai_object_id_t vid;
     sai_deserialize_object_id(strVid, vid);
 
     sai_object_id_t rid;
 
     if (!m_translator->tryTranslateVidToRid(vid, rid))
     {
-        SWSS_LOG_WARN("port VID %s, was not found (probably port was removed/splitted) and will remove from counters now",
-                sai_serialize_object_id(vid).c_str());
-
-        op = DEL_COMMAND;
+        if (operateDb)
+        {
+            SWSS_LOG_THROW("port VID %s, was not found (probably port was removed/splitted) and will remove from counters now",
```

Comment:
Why we need to throw  here? It will crash, and fail in remove counter from. Flex list is not critical and syncd can stuull run, please remove

Reply:
removed.